recipes/ohpipeline/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeDeps, CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import copy, get, export_conandata_patches, apply_conandata_patches
+from conan.tools.apple import is_apple_os
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class OhPipelineConan(ConanFile):
+    name = "ohpipeline"
+    description = "OpenHome audio pipeline"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/openhome/ohPipeline"
+    topics = ("openhome", "ohnet", "ohpipeline", "upnp")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=os.path.join(self.export_sources_folder, "src"))
+        export_conandata_patches(self)
+
```

Comment:
```suggestion

    def build_requirements(self):
        self.tool_requires("cmake/[>3.XX <4]")

```

Let's install cmake directly as tool requirement. Don't worry, users still can manage it by their profile.